infrastructure_files/getting-started-with-zitadel.sh

```diff
@@ -527,6 +624,10 @@ renderCaddyfile() {
     }
 }
 
+$NETBIRD_DOMAIN {
```

Comment:
What is the effect of this setting? Does caddy will look for the certificates if none is provided? Does it fail? I think this block should be optional.